### DISCUSSION TITLE
Using Gmail as provider to send emails

### DISCUSSION THREAD

**Discussion by JesusMtnez** [firefly-iii/firefly-iii#7003] [Firefly III] [answered]
In 2022, Google disabled using username/password to authorize third-party applications. Is still possible to configure gmail as provider so firefly could send emails using it? I have not been able to see any reference to it.

Thank you!

**Comment by JC5** [comment, marked as answer]
Yes, you'll have to create a specific app password for Firefly III and use that to authenticate.
